packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -145,21 +145,23 @@ export class SequentialHandler {
 	public async queueRequest(routeId: RouteData, url: string, options: RequestInit, body?: unknown): Promise<unknown> {
 		// Wait for any previous requests to be completed before this one is run
 		let queue = this.#asyncQueue;
-		// Separate sublimtted requests when already sublimitted
-		let sublimitted: boolean | null = this.#sublimitedQueue && isSublimitted(routeId.bucketRoute, body);
-		if (sublimitted) {
+		// Separate sublimited requests when already sublimited
+		let sublimited: boolean | null = this.#sublimitedQueue && isSublimited(routeId.bucketRoute, body);
```

Comment:
`hasSublimit` maybe?

Reply:
I was reserving that for checking if the route itself has a sublimit if there are any more in the future, but I think that could just be an array in util we could .includes against.

Reply:
I just think `isSublimited` sounds very odd but I guess it won't matter too much